libmscore/note.cpp

```diff
@@ -3442,7 +3442,7 @@ Shape Note::shape() const
       Shape shape(r);
       for (NoteDot* dot : _dots)
             shape.add(symBbox(SymId::augmentationDot).translated(dot->pos()));
-      if (_accidental)
+      if (_accidental && _accidental->addToSkyline())
```

Comment:
FWIW, I wonder if we could lose this NDEBUG duplication?  Right now it buys us some better debug info on the shapes when compiling in debug mode, without a performance penalty otherwise.  But maybe there is a less error-prone way of doing that?  Way too easy for these two sections of code to get out of sync.